File: lessons/compiled_lang/cs/src/k10.parameters.cs
using System;

class K10Parameters
{
    static void AddNums(params int[] numbers)
    {
        int sum = 0;
        foreach (int num in numbers)
        {
            sum += num;
        }
        Console.WriteLine("The summation is: " + sum + ".");
    }

    static void Main()
    {
        Console.WriteLine("Sending: 5, 2, 4, 3, 6");
        AddNums(5, 2, 4, 3, 6);
    }
}

File: lessons/compiled_lang/cpp/src/k10.parameters.cpp
#include <initializer_list>
#include <iostream>

void addNums(std::initializer_list<int> numbers) {
    int sum = 0;
    for (int num : numbers) {
        sum += num;
    }
    std::cout << "The summation is: " << sum << "." << std::endl;
}

int main() {
    std::cout << "Sending: 5, 2, 4, 3, 6" << std::endl;
    addNums({5, 2, 4, 3, 6});

    return 0;
}
